interpreter/terminal_interface/terminal_interface.py

```diff
@@ -151,12 +151,16 @@ def terminal_interface(interpreter, message):
 
                             scan_code(code, language, interpreter)
 
+                        interpreter.run_code_promt = False
                         response = input(
                             "  Would you like to run this code? (y/n)\n\n  "
                         )
                         print("")  # <- Aesthetic choice
 
                         if response.strip().lower() == "y":
+                            interpreter.run_code_promt = True
+
+                        if interpreter.run_code_promt:
```

Comment:
I think this is an unnecessary condition.

We can just set the property to true and follow the code path in this condition without needing to check the value of the property we just set.

Reply:
There is a third one in response.py

Reply:
Yeah, for that we need to set the `interpreter.run_code_promt` (which I just noticed has a typo, can we make that `interpreter.run_code_prompt`), but we don’t need this `if` statement because at this point in the code that property can only have one value because we just set it a couple of lines above.

Reply:
I added it to stop it running the code regardless of y/n. It just didn't output the result when saying n. 
I'll double check and come back to this after work 👍